presto-common/src/main/java/com/facebook/presto/common/predicate/Range.java

```diff
@@ -244,13 +244,27 @@ public String toString(SqlFunctionProperties properties)
     {
         StringBuilder buffer = new StringBuilder();
         if (isSingleValue()) {
-            buffer.append('[').append(low.getPrintableValue(properties)).append(']');
+            buffer.append("[\"").append(low.getPrintableValue(properties).toString().replace("\"", "\\\"")).append("\"]");
```

Comment:
the logic of adding quotes is repeated multiple times; consider adding a helper method that takes a buffer and a value to add and adds the value with quotes.

Reply:
done